docs/en_US/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Report Bugs
+lang: en_US
+sidebarDepth: 2
+---
+
+# Report bugs
+
+## Guidelines
+Here is a small tutorial about how to optimally report a bug.
+
+Before reporting a bug, check the wiki to see if it a desired feature, or an already reported bug.
+
+::: tip
+When reporting a bug, describe it in as much detail as possible. The more detailed the report, the faster we can help you.
+:::
+::: warning
+Please note that we do not officially support; older browsers; mobile browsers; Microsoft Edge; or Internet Explorer.
```

Comment:
Again the ";" stuff. Is this intended?

Reply:
See https://github.com/LSS-Manager/LSSM-V.4/pull/155/files#r511679383